packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingInvoiceDetails.tsx

```diff
@@ -63,6 +63,7 @@ const StyledMenu = styled((props: MenuProps) => {
         vertical: 'top',
         horizontal: 'right',
       }}
+      container={props.container}
```

Comment:
@vkuprin, here you must just pass `useRootElements().root` instead of `props.container`.
There is no chance that `StyledMenu` will receive the `container` property, so, just force it as:
```
container={useRootElements().root}
```

Reply:
Nope, I'm wrong, it will receive property from `<StyledMenu {...menuProps}>`.
Anyway, I would add the default property as a fallback. The `menuProps` will override it for sure too.